tests/kubeconfig

```diff
@@ -0,0 +1,19 @@
+    server: https://api.croberts3.dev.datahub.redhat.com:6443
```

Comment:
@crobby  Did you really want to commit this file?

Reply:
oh jeez